### ISSUE TITLE
RBCHandover and InformationFilter Collaboration

### ISSUE THREAD

**Issue by UweSteinkeFromSiemens** [openETCS/modeling#430]
Discussion results between @astante and @UweSteinkeFromSiemens: 

- The **RBC/RBC handover** procedure manages the OBU transition process from the (current) handing over RBC to the accepting RBC. If the OBU is equipped with 2 mobiles, the OBU during the handover phase maintains 2 radio communication sessions with both RBCs in parallel. The RBCHandover therefore controls the session mangement with 2 MoRC (Management of Radio Communication) instances. 

- During the handover process, the supervision is transferred from the handing over RBC to the accepting RBC. 

- The **InformationFilter** is - besides others - in charge of 
  - letting pass all relevant datagrams from the **supervising RBC**
  - letting pass all datagrams from the **non-supervising RBC** relevant for radio connection, radio session management and RBC handover process.
  - buffering all relevant datagrams from the accepting RBC as required in Subset-026, 3.15.1.3.6.

- The InformationFilter needs to know, which RBC is the supervising RBC. The **RBCHandover** provides this information as an output comprising NID_C and NID_RBC of the supervising RBC. The supervision transfer can be recognized by detecting a change in NID_C / NID_BG. 

- Since the InformationFilter operates on both input data streams, from the handing over and the accepting RBC, the 2 data streams from both RBC have to be merged into one stream at the input side (before they are forwarded to Manage_TrackSideInformation_Integration).

- Consumers of the InformationFilter output data stream have to consider that both RBC input streams have been merged. 

Ref. to #341.

**Comment by UweSteinkeFromSiemens** [comment]
The main function RBCHandover in https://github.com/openETCS/modeling/tree/master/model/Scade/System/ObuFunctions/Radio/Handover/RBC_Handover has got a new output "supervisingRBC" indicating the currently active RBC.

**Comment by UweSteinkeFromSiemens** [comment]
As I just recognized, NIC_C and NID_RBC - making up the RBD ID - are not part of radio track-to-train messages. It's incomprehensible how the InformationFilter is able to differentiate radio messages from different RBCs, if the RBC source information is not available in the datagram.  

@astante: Can you please explain?

**Comment by BaseliyosJacob** [comment]
Needs to be solved with @astante and @berndhekele

**Comment by christianstahl** [comment]
The current version of the information filter takes now the RadioStatus1 as an input. Still, the input Supervising device is a constant set to 0 in the EVC model.
